src/components/EditListingWizard/EditListingWizard.js

```diff
@@ -116,9 +116,9 @@ const tabCompleted = (tab, listing) => {
     case ABOUTTHISSERVICE:
       const { averageTurnAroundTime, explainMore } = publicData
       return !!averageTurnAroundTime && !!explainMore
-    case PAYMENT:
-      const { payPalEmail, phoneNumber } = publicData
-      return !!payPalEmail && !!phoneNumber
+    case CONTACT:
+      const { phoneNumber } = publicData
+      return !!phoneNumber
```

Comment:
Can we validate that this is a real phone number before we let them move past that form? I don't care where we do it, here or in the input itself.

Reply:
It will be in next PR